ansible/csm_packages.yml

```diff
@@ -23,21 +23,19 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 # Compute Nodes Play
-- hosts: Compute,Application
-  gather_facts: yes
+- hosts: Compute:Application
```

Comment:
Why change from comma to colon? Ansible's docs show both as being valid.

Multiple hosts | host1:host2 (or host1,host2)

Reply:
Just a consistency thing; I crafted the correct logic in one playbook, and once I had the right selection of hosts, I pasted it in all affected playbooks and then just swapped group names.